Hallo.Test/Hallo.Test.csproj

```diff
@@ -16,7 +16,7 @@
     </ItemGroup>
 
     <ItemGroup>
-      <ProjectReference Include="..\Hallo\Hallo.csproj" />
+      <ProjectReference Include="..\Hallo.AspNetCore.Mvc\Hallo.AspNetCore.Mvc.csproj" />
```

Comment:
We could probably create a Hallo.Test.Mvc project just to test the outputformatter but figured I'd leave them as they are